scrum/fileconfigstorage.go

```
@@ -42,7 +40,7 @@ func (configStorage *FileConfigurationStorage) Save(config *Config) {
 		return
 	}
 
-	io.Writer(file).Write(buffer.Bytes())
+	_, _ = io.Writer(file).Write(buffer)
```

Why assign nothing?

---

I had a warning in GoLand because the error was not handled. I'll handle it instead of just hiding the error ;)